src/proposals/20231129/SparkEthereum_20231129.t.sol

```diff
@@ -0,0 +1,174 @@
+        assertEq(daiConfigAfter.interestRateStrategy, NEW_DAI_INTEREST_RATE_STRATEGY);
+
+        _validateDaiInterestRateStrategy(
+            daiConfigAfter.interestRateStrategy,
+            NEW_DAI_INTEREST_RATE_STRATEGY,
+            DaiInterestStrategyValues({
+                vat:                oldStrategy.vat(),
+                pot:                oldStrategy.pot(),
+                ilk:                oldStrategy.ilk(),
+                baseRateConversion: oldStrategy.baseRateConversion(),
+                borrowSpread:       oldStrategy.borrowSpread(),
+                supplySpread:       NEW_SUPPLY_SPREAD,
+                maxRate:            oldStrategy.maxRate(),
+                performanceBonus:   oldStrategy.performanceBonus()
+            })
+        );
+    }
+
+    function test_assetLiabilityDiff() public {
+        uint256 startingTimestamp = block.timestamp;
+
+        uint256 startDiff = _getAssetLiabilityDiff(DAI);
+
+        assertEq(startDiff, 221_232.428488870663797097 ether);
+
+        // Take snapshot before simulating protocol activity
+        uint256 snapshot = vm.snapshot();
+
+        // Simulate 100 days of protocol activity
+        for (uint256 i = 0; i < 2400; i++) {
+            vm.warp(startingTimestamp + i * 1 hours);
+            _supply(DAI, 1e18);
+        }
+
+        // Demonstrate that the asset liability diff would continue to increase over time
+        assertEq(_getAssetLiabilityDiff(DAI),             778_319.620039363706458871 ether);
+        assertEq(_getAssetLiabilityDiff(DAI) - startDiff, 557_087.191550493042661774 ether);
+
+        // Warp back to original timestamp and snapshot
+        vm.warp(startingTimestamp);
+        vm.revertTo(snapshot);
+
+        // Execute spell
+        GovHelpers.executePayload(vm, payload, executor);
+
+        ( uint256 supplyRate1, uint256 variableBorrowRate1 ) = _getRates(DAI);
+
+        // Diff should be small since utilization is almost 100%
+        assertEq(supplyRate1,         0.046330401155582341410874849e27);
+        assertEq(variableBorrowRate1, 0.053790164207174267760128000e27);
+
+        // Supply a small amount of DAI to update rates
+        _supply(DAI, 1e18);
+
+        ( uint256 supplyRate2, uint256 variableBorrowRate2 ) = _getRates(DAI);
+
+        // Show that rates have updated
+        // Diff is small since utilization is almost 100%
+        assertEq(supplyRate2,         0.051078367754112897710958380e27);
+        assertEq(variableBorrowRate2, 0.053790164207174267760128000e27);
+
+        // Supply rate change is slightly less than supplySpread update because of utilization
+        assertEq(variableBorrowRate2 - variableBorrowRate1, 0);
+        assertEq(supplyRate2 - supplyRate1,                 0.004747966598530556300083531e27);
+
+        // Make sure that diff is the same as start of test
+        assertApproxEqAbs(_getAssetLiabilityDiff(DAI), startDiff, 1);
+
+        // Simulate 100 days of protocol activity
```

Comment:
Wouldn't it be better to simulate 365 days? This should give more intuitively interpretable results for anyone checking the numbers without checking the code, or performing exact calculations.

Reply:
Updated to a year